SegmentEditorFloodFilling/SegmentEditorFloodFillingLib/SegmentEditorEffect.py

```diff
@@ -36,7 +36,7 @@ def helpText(self):
 
   def activate(self):
     # Update intensity range
-    self.masterVolumeNodeChanged()
+    self.sourceVolumeNodeChanged()
```

Comment:
Keep backward compatibility with Slicer-5.0 to make it easier to keep the extension up-to-date for both stable and preview releases.